icu4j/main/classes/core/src/com/ibm/icu/util/HebrewCalendar.java

```diff
@@ -62,7 +62,7 @@
  * <p>
  * This class should not be subclassed.</p>
  * <p>
- * HebrewCalendar usually should be instantiated using 
+ * HebrewCalendar usually should be instantiated using
```

Comment:
I touched this java file for removing redandant cast. OpenJDK javac produces warnings for them. Because we also set up eclipse project to clean up source when saved, there are many lines showing up in diff viewer, just because space changes (trailing spaces are removed - tab is replaced with 4 spaces...)